modules/openapi-generator/src/main/resources/typescript-rxjs/runtime.mustache

```diff
@@ -121,11 +115,14 @@ export class BaseAPI {
      * Create a shallow clone of `this` by constructing a new instance
      * and then shallow cloning data members.
      */
-    private clone = <T extends BaseAPI>(): T => 
```

Comment:
Why was this change made?

Reply:
I think I saw now point in having this generic there. 
Now I can see that people might want to extend `BaseAPI` and it won't work then? Perhaps the generic could be avoided by using `private clone = (): this =>`?

Reply:
Yes, it should be `this` in the return type, because without `clone` and inheriting methods are not polymorphic.